src/Upgrade/Application/Service/UpgradeService.php

```diff
@@ -42,6 +50,12 @@ public function upgrade(): StepsResponseDto
     {
         $strategy = $this->strategyResolver->getStrategy($this->configurationProvider->getUpgradeStrategy());
 
-        return $strategy->upgrade();
+        $stepsResponse =  $strategy->upgrade();
```

Comment:
FILE: ...ader/upgrader/src/Upgrade/Application/Service/UpgradeService.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 53 | ERROR | [x] Duplicate spaces at position 25.
    |       |     (SlevomatCodingStandard.Whitespaces.DuplicateSpaces.DuplicateSpaces)
 56 | ERROR | [x] A cast statement must not be followed by a
    |       |     space
    |       |     (Generic.Formatting.NoSpaceAfterCast.SpaceFound)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 47.68 secs; Memory: 36MB

Script phpcs handling the cs-check event returned with error code 2
Error: Process completed with exit code 2.